verify-cborld-presentation/.eslintrc

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
Assume this file is auto generated? Then we don't touch :)

Reply:
Have updated with some rules from mobile-wallet